src/platform/qt/library/LibraryController.h

```diff
@@ -15,11 +15,17 @@
 
 #include <mgba/core/library.h>
 
+class QTreeView;
+class QListView;
+class QAbstractItemView;
+class QSortFilterProxyModel;
+
 namespace QGBA {
 
 // Predefinitions
 class LibraryGrid;
 class LibraryTree;
+class LibraryModel;
```

Comment:
Nit: These weren't in alphabetical order before, but you can fix that since you're here. Also, that comment seems useless.